src/ssh/azext_ssh/ssh_info.py

```diff
@@ -86,7 +88,7 @@ class ConfigSession():
     def __init__(self, config_path, resource_group_name, vm_name, ssh_ip,
                  public_key_file, private_key_file, overwrite, use_private_ip,
                  local_user, cert_file, port, resource_type, credentials_folder,
-                 ssh_proxy_folder, ssh_client_folder, yes_without_prompt):
+                 ssh_proxy_folder, ssh_client_folder, yes_without_prompt, resource_tag=None):
```

Comment:
Why does this resource_tag argument has a default value of None and the one in SSHSession doesn't? Just a consistency thing